src/editor/components/LessonSection.js

```diff
@@ -1,32 +1,177 @@
-import React from 'react';
+import React, { useState } from 'react';
 import LessonPlanTextInput from './LessonPlanTextInput';
-import { Text, View, SafeAreaView, StyleSheet } from 'react-native';
+import { Text, View, SafeAreaView, Platform, StyleSheet, TextInput } from 'react-native';
+import { useDispatch } from 'react-redux';
+import {
+  addToSection,
+  removeFromSection,
+} from '../../services/editor/lessonPlanSlice';
+import store from '../../services/configureStore';
+
+const LessonSection = ({ sectionType, subtitle }) => {
+  const [sectionContent, setSectionContent] = useState([]);
+  // const sectionContent = [
+  //   {
+  //     section: "warmup",
+  //     content: "warmup content",
+  //   }
+  // ]
+  const [sectionActivityCards, setSectionActivityCards] = useState([]);
+  const [clicked, setClicked] = useState(false);
+  const dispatch = useDispatch();
+  const handleClick = () => {
+    setClicked(true);
+  };
+
+  const ContentCard = () => {
+    return (
+      clicked && (
+        <View style={styles.ContentCardStyle}>
+          <TextInput
+            multiline
+            placeholder={'Add Text'}
+            returnKeyType="next"
+            onSubmitEditing={e => {
+              if (e.nativeEvent.text) {
+                setSectionContent([...sectionContent, e.nativeEvent.text]);
+                store.dispatch(
+                  addToSection({
+                    type: 'text',
+                    section: sectionType,
+                    content: e.nativeEvent.text,
+                  })
+                );
+              }
+              setClicked(false);
```

Comment:
```suggestion
              setClicked(false); // Close the box once user is done
```
Adding more comments overall would help